Rmd/Sample_QC.Rmd

```diff
@@ -144,18 +144,18 @@ if (params$collapse_replicates == T) {
                             names_to = c('.value','flowcell'),
                             names_pattern = '(.*)_(.*)')
   df[is.na(df)] <- 0
-  sampleDataTable <- data.table(df %>% dplyr::select(-c(flowcell)))
-  results <- sampleDataTable[, lapply(.SD,sum), by = "X"]
+  count_dataTable <- data.table(df %>% dplyr::select(-c(flowcell)))
+  results <- count_dataTable[, lapply(.SD,sum), by = "X"]
```

Comment:
Since `count_dataTable` is just used within this function can we call it something else to distinguish it from `count_data_file` and `count_data` since it's just a table of samples? `sample_data` or something.